mirrord/layer/src/outgoing/tcp.rs

```diff
@@ -22,6 +28,9 @@ use tracing::{error, info, trace, warn};
 use super::*;
 use crate::{common::ResponseDeque, detour::DetourGuard, error::LayerError};
 
+/// For unix socket addresses, relative to the temp dir (`/tmp/mirrord-bin/...`).
+pub const MIRRORD_UNIX_STREAMS: &str = "mirrord-unix-sockets";
```

Comment:
Try finding a better name for it (even just affix _DIR)

Reply:
```suggestion
pub const UNIX_STREAMS_DIRNAME: &str = "mirrord-unix-sockets";
```